apps/src/code-studio/assets/SelectStartAnimations.jsx

```diff
@@ -0,0 +1,160 @@
+
+    let propsByKey = {...this.state.propsByKey};
+    delete propsByKey[key];
+    this.setState({propsByKey: propsByKey});
+  };
+
+  displaySelectedSprites = () => {
+    const {propsByKey, orderedKeys} = this.state;
+    return orderedKeys.map(key => {
+      return (
+        <img
+          key={key}
+          src={propsByKey[key].sourceUrl}
+          alt={propsByKey[key].name}
+          style={styles.thumbnail}
+          role="button"
+          onClick={() => this.removeAnimationFromList(key)}
+        />
+      );
+    });
+  };
+
+  render() {
+    const {orderedKeys, propsByKey} = this.state;
+    return (
+      <div>
+        <h2>Select Starting Animations</h2>
+        <div style={styles.pageBreak}>
+          <h3>Selected Animations:</h3>
+          {this.displaySelectedSprites()}
+        </div>
+        {this.props.useAllSprites && (
+          <div style={styles.pageBreak}>
+            <h3>
+              Hidden Animations (Animations that don't normally appear in the
+              animation library):
+            </h3>
+            <AnimationPickerBody
+              is13Plus={true}
+              onDrawYourOwnClick={() =>
+                console.log('Not supported at this time')
+              }
+              onPickLibraryAnimation={this.addAnimationToList}
+              onAnimationSelectionComplete={() => {}}
+              onUploadClick={() => console.log('Not supported at this time')}
+              playAnimations={false}
+              libraryManifest={this.state.levelAnimationsManifest}
+              hideUploadOption={false}
+              hideAnimationNames={false}
+              navigable={true}
+              hideBackgrounds={false}
+              canDraw={false}
+              pickerType={PICKER_TYPE.spritelab}
+              selectedAnimations={[]}
+            />
+          </div>
+        )}
+        <div style={styles.pageBreak}>
+          <h3>Library Animations:</h3>
+          <AnimationPickerBody
+            is13Plus={true}
+            onDrawYourOwnClick={() => console.log('Not supported at this time')}
+            onPickLibraryAnimation={this.addAnimationToList}
+            onAnimationSelectionComplete={() => {}}
+            onUploadClick={() => console.log('Not supported at this time')}
+            playAnimations={false}
+            libraryManifest={this.state.libraryManifest}
+            hideUploadOption={false}
+            hideAnimationNames={false}
+            navigable={true}
+            hideBackgrounds={false}
+            canDraw={false}
+            pickerType={PICKER_TYPE.spritelab}
+            selectedAnimations={[]}
+          />
+        </div>
+        <p>{JSON.stringify({orderedKeys, propsByKey})}</p>
+      </div>
+    );
+  }
+}
+
+const styles = {
+  thumbnail: {
+    height: THUMBNAIL_SIZE,
+    borderStyle: 'solid',
+    borderColor: color.light_gray,
+    borderWidth: THUMBNAIL_BORDER_WIDTH,
+    borderRadius: 12,
+    cursor: 'pointer',
+    ':hover': {
```

Comment:
this functionality comes from radium, but we aren't wrapping this component in the radium higher-order function